source/game/g_svcmds.cpp

```diff
@@ -117,9 +118,16 @@ static void Cmd_Match_f( void )
 
 typedef struct
 {
-	unsigned mask;
-	unsigned compare;
+	union {
+		uint64_t steamid;
+		struct {
+			unsigned mask;
+			unsigned compare;
+		};
```

Comment:
This is an ip address (+ range) in disguise.

Reply:
are you saying I should rename it?

Reply:
probably a type or something less obfuscated.